site/content/en/docs/Reference/agones_crd_api_reference.html

```diff
@@ -7319,7 +7320,7 @@ <h3 id="MultiClusterSetting">MultiClusterSetting
 <hr/>
 <h2 id="autoscaling.agones.dev">autoscaling.agones.dev</h2>
 <p>
-<p>Package v1alpha1 is the v1alpha1 version of the API.</p>
+<p>Package v1 is the v1 version of the API.</p>
```

Comment:
Should I wrap these changes in the `{{% feature ... %}}` blocks like I did for the other documentation updates? It's unclear since this change was auto-generated.

Reply:
No, they should be wrapped automatically. I mean we wrap the whole block, and have current and previous version as a whole piece so that's easier to compare. I have commented with the algorithm in my PR, fixed gen_api_docs a bit.

Reply:
I've rebased on top of your PR and will re-generate this file.

Reply:
I re-ran `make gen-api-docs` after rebasing on top of your PR but this file doesn't look any different.... So my original question still stands about whether I should be wrapping this in the `{{% feature ... %}}` blocks.

Reply:
I just tested this via `make site-server` and then `make site-server ENV=RELEASE_VERSION=0.11.0` - and the former shows the v1 version, and the latter shows the v1alpha1 version - so I'm not 100% sure what the issue here is?